docs/README.md

```diff
@@ -1 +1,116 @@
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+### `block-2` props
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+Prop types are: 
+
+- `string` 
+- `enum` 
+- `number` 
+- `boolean` 
+- `object` 
+- `array` 
+
+When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
+
+- `propName` object:
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 
+
+## Modus Operandi *(not mandatory)*
+
+There are scenarios in which an app can behave differently in a store, according to how it was added to the catalog, for example. It's crucial to go through these **behavioral changes** in this section, allowing users to fully understand the **practical application** of the app in their store.
+
+If you feel compelled to give further details about the app, such as it's **relationship with the VTEX admin**, don't hesitate to use this section. 
+
+## Customization
+
+The first thing that should be present in this section is the sentence below, showing users the recipe pertaining to CSS customization in apps:
+
+`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`
+
+Thereafter, you should add a single column table with the available CSS handles for the app, like the one below. Note that the Handles must be ordered alphabetically.
+
+| CSS Handles |
+| ----------- | 
+| `XXXXX` | 
+| `XXXXX` | 
+| `XXXXX` | 
+| `XXXXX` | 
+| `XXXXX` |
+
+
+If there are none, add the following sentence instead:
+
+`No CSS Handles are available yet for the app customization.`
```

Comment:
This is an instruction to add customization info to the doc! If no Handles are currently available, this part should be removed - only keep the `No CSS Handles are available yet for the app customization`.